CourseApp/ConsoleInputOutput.cs

```diff
@@ -0,0 +1,79 @@
+namespace CourseApp
+{
+    using System;
+    using CourseApp.Class;
+
+    public class ConsoleInputOutput
+    {
+        public void StartProgram()
+        {
+            Console.Clear();
+            Console.WriteLine("Hello!\nThis program can create files with default or your params\nFile with default params - 1 | File with your params - 2\nWhat you want?");
```

Comment:
this one may not work on cross platform Console has newLine

Reply:
Fixed